extensions/amp-analytics/0.1/vendors.js

```diff
@@ -2094,7 +2092,13 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
     },
   },
 
+  'bg': {
+    'transport': {
+      'iframe': 'https://tpc.googlesyndication.com/b4a/b4a-runner.html',
```

Comment:
I'm fine with this, but doesn't it contradict previous concerns that you and/or @keithwrightbos had about file size, because now we will pull in all of vendors.js at times when only iframe transport info is needed?

Reply:
Not very sure about the file size concern. Can you elaborate?

Reply:
nvm, the test failure explained what you meant. amp-ad-exit is importing this deleted file. I'll fix.